kie-server-parent/kie-server-services/kie-server-services-openshift/src/test/java/org/kie/server/services/openshift/impl/storage/cloud/KieServerStateOpenShiftRepositoryTest.java

```diff
@@ -60,8 +65,8 @@
     protected OpenShiftClient client;
     protected KieServerStateOpenShiftRepository repo;
 
-    @Rule
-    public OpenShiftServer server = new OpenShiftServer(false, true);
+    // Need to use direct OpenShiftMockServer instead of OpenShiftServer as HTTPS support needs to be disabled for proper functionality in JDK 11
+    public OpenShiftMockServer server = new OpenShiftMockServer(new Context(), new MockWebServer(), new HashMap<ServerRequest, Queue<ServerResponse>>(), new KubernetesCrudDispatcher(), false);
```

Comment:
@sutaakar Initially, I think that TLS has already been turned off until found this:
https://github.com/fabric8io/kubernetes-client/issues/1243
Not sure why it has not been fix for over years. Looks like nobody care.